components/button/README.md

```diff
@@ -198,7 +166,21 @@ The `d2l-button-add` element is for quickly adding new items inline (for example
 
 ## Floating Buttons [d2l-floating-buttons]
 
-See [floating buttons](https://github.com/BrightspaceUI/core/tree/main/components/button/floating-buttons.md).
+See [floating buttons](../../components/floating-buttons).
```

Comment:
```suggestion
See [floating buttons](./floating-buttons.md).
```

You'll want to test this but I think it will work here and in Daylight. The tooltip one below looks to be fine.

Reply:
Okay will test it out! Thx